src/coreclr/vm/threads.cpp

```diff
@@ -1124,7 +1124,11 @@ extern "C" void *JIT_WriteBarrier_Loc;
 
 #ifdef TARGET_ARM64
 extern "C" void (*JIT_WriteBarrier_Table)();
+
+extern void *JIT_WriteBarrier_Loc;
```

Comment:
I think this needs to be extern "C".

Reply:
Right, I am sorry for missing that when reviewing the change.